### PR TITLE
Added support for forwarded IP addresses (behind (ha)proxy)

### PR DESCRIPTION
**Pull request by L00Cyph3r**
Since there was no way of setting the request-context outside of this library I figured that this would be the right place to support proxy/loadbalancer-setups.
Shoot me if anything.

### CODE REVIEW

**Comment by philkra** [comment]
@L00Cyph3r  great! thanks a lot. but would you please change the PR to the `development` branch?

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name" : "philkra/elastic-apm-php-agent",
+    "name" : "l00cyph3r/elastic-apm-php-agent",
```
Comment (dstepe) [inline composer.json]: I don't think this change should be part of the PR.

Comment (philkra) [reply composer.json]: yeah, I saw that and would have issued a change request once this pr is against the dev branch. that will break composer. thank for your keen eyes :)

**Comment by philkra** [comment]
please select development branch for this pr

File: composer.json (hunk 1 above)
Comment (L00Cyph3r) [reply composer.json]: Whoops, I issued the PR and I changed this afterwards so I could use this in my project without waiting for the PR-merge. Didn't know any future changes would come through as well

Comment (dstepe) [reply composer.json]: @L00Cyph3r I typically use a composer vcs repo type and just point my fork in cases like that. It's not necessary to change the composer.json file that way.

https://getcomposer.org/doc/05-repositories.md#loading-a-package-from-a-vcs-repository

**Pull request closed without merging**